docs/sphinx/source/whatsnew/v0.9.6.rst

```diff
@@ -11,7 +11,7 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
-
+* Lakshya Garg (:ghuser:`Lakshyadevelops` :pull:`1705`)
```

Comment:
This isn't quite how we do "whatsnew" entries.  This line should be a description of the new feature, something like this:
```
* Added a new irradiance decomposition model :py:func:`pvlib.irradiance.louche`. (:pull:`1705`)
```

And then your name and GH username go in the `Contributors` section at the bottom of this file.  Check the `0.9.5.rst` file for reference.